AutoCollection/ServerRequestParser.ts

```diff
@@ -110,6 +106,10 @@ class ServerRequestParser extends RequestParser {
         return this.requestId;
     }
 
+    public getCorrelationContext() {
+        return this.correlationContext;
```

Comment:
This could get removed entirely since it should be CorrelationContext.customProperties